src/course-home/outline-tab/SequenceLink.jsx

```diff
@@ -115,6 +116,9 @@ const SequenceLink = ({
           </div>
         </div>
         <div className="row w-100 m-0 ml-3 pl-3">
+          <small className="text-body pl-2 pr-0">
+            {optional ? intl.formatMessage(messages.optionalCompletion) : ''}
+          </small>
```

Comment:
Let's move this to a `Badge`. The text here is related to grading, which is a separate mechanism.

Reply:
Done. I added it as small text because I gelt the Badge overloads the webpage a bit. But maybe it's fine:
![image](https://github.com/open-craft/frontend-app-learning/assets/19278837/799de93a-2390-4c52-b55f-a57e3a896b30)